presto-main/src/main/java/com/facebook/presto/operator/TableFinishOperator.java

```diff
@@ -14,6 +14,7 @@
 package com.facebook.presto.operator;
 
 import com.facebook.presto.Session;
+import com.facebook.presto.execution.Lifespan;
```

Comment:
I would move the changes to this class to the `Add operator execution info for TableWriterOperator and DeleteOperator ` as much as possible.